app/controllers/user_journey_controller.rb

```diff
@@ -75,46 +75,51 @@ def submit
   end
 
   def confirm
-    if @certificate.component.enabled_signing_certificates.count >= 2 && @certificate.signing?
+    if @certificate.signing? && @certificate.component.enabled_signing_certificates.count >= 2
```

Comment:
Do we not want to check the event is valid first, and then put the rest of the if/elsif statements within? Or I guess it doesn't matter because it will throw an error if the first condition is true anyway. Other than Jakubs comments all looks good to me :)